### ISSUE TITLE
Don't parse URLs or characters that are encoded in a URL

### ISSUE THREAD

**Issue by KevinNovak** [wanasit/chrono#393]
**Issue**
Characters that are in a URL can be parsed by Chrono and recognized as dates/times.

**Background**:
I noticed that for this specific URL:
`https://tenor.com/view/%e3%83%89%e3%82%ad%e3%83%89%e3%82%ad-%e7%b7%8a%e5%bc%b5-%e5%a5%bd%e3%81%8d-%e3%83%8f%e3%83%bc%e3%83%88-%e5%8f%af%e6%84%9b%e3%81%84-gif-15876325`
Which in it's unencoded form is the following:
`https://tenor.com/view/ドキドキ-緊張-好き-ハート-可愛い-gif-15876325`

More specifically this character: `緊` becomes encoded as `%e7%b7%8a`.

The encoded form of this URL contains "%8a", which gets recognized as 8 AM by Chrono.

**Solution**
So I suppose this is an enhancement request for Chrono to ignore URLs completely, or to ignore encoded characters that are in a URL.

**Comment by wanasit** [comment]
Thanks for reporting this. I fixed the pattern like "8a" in 77fbd83.

I don't know if we should ignore URLs completely, as there are some URL cases that have dates in them and I think we should continue support parsing them e.g.:
```
https://somesite.com/article/2021/08/12/acb-acc.html
https://somesite.com/article/acb-2021-aug-12.html
```

**Comment by KevinNovak** [comment]
Thank you for the fix! Ignoring words with "a" and "p" probably also solves some other false positives as well.

> I don't know if we should ignore URLs completely

Agreed, I think this it is easy enough for a developer to just strip out URLs before using chrono to parse.
